### ISSUE TITLE
Use powershell to update path variables on windows

### ISSUE THREAD

**Issue by Mehgugs** [luarocks/luarocks#1112]
The command:
```
powershell -command "[System.Environment]::SetEnvironmentVariable('LUA_PATH', '<VALUE>',[System.EnvironmentVariableTarget]::User)"
```
Will permanently install the environment variable for the current user. It is much better than SET and you can select different targets easily.

**Comment by hishamhm** [comment]
Thank you for the suggestion! What's the idea, to make that the output of `luarocks path`?

**Comment by Mehgugs** [comment]
Yes, I believe it would make the process of getting a windows user's env variables set up a bit easier. I tested it and the User target can be ran without an elevated shell.

**Comment by hishamhm** [comment]
Nice! For compatibility sake (and consistency with Unix where these settings are not permanent), I think this should be activated through a flag, e.g, `--persist`
